components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -159,13 +214,14 @@ - (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
   XCTAssertFalse(self.manager.internalManager.overlayView.accessibilityViewIsModal);
 }
 
-- (void)testSnackbarSetAccessibiltyViewIsModalShouldBeNoForActionSnacbarsWhenManagerIsNo {
+- (void)testSnackbarAccessibiltyViewIsModalWithSnackbarViewOverwriteWithActions {
   // Given
+  self.delegate.shouldSetSnackbarViewAccessibilityViewIsModal = YES;
   self.manager.internalManager.isVoiceOverRunningOverride = YES;
   MDCSnackbarMessageAction *action = [[MDCSnackbarMessageAction alloc] init];
   action.title = @"Tap Me";
   self.message.action = action;
-  self.manager.shouldEnableAccessibilityViewIsModal = NO;
+  self.message.accessibilityViewIsModal = YES;
```

Comment:
If this work, then shouldn't this be the preferred mechanism for clients, rather than relying on the delegate?